core/common/src/test/java/alluxio/ConfigurationRuleTest.java

```diff
@@ -32,4 +32,16 @@ public void evaluate() throws Throwable {
     new ConfigurationRule(ImmutableMap.of(PropertyKey.MASTER_ADDRESS, "testValue"))
         .apply(statement, null).evaluate();
   }
+
+  @Test
+  public void changeConfigurationForDefaultNullValue() throws Throwable {
+    Statement statement = new Statement() {
+      @Override
+      public void evaluate() throws Throwable {
+        Assert.assertEquals("testValue", Configuration.get(PropertyKey.SECURITY_LOGIN_USERNAME));
+      }
+    };
+    new ConfigurationRule(ImmutableMap.of(PropertyKey.SECURITY_LOGIN_USERNAME, "testValue"))
+        .apply(statement, null).evaluate();
```

Comment:
It would be good to also check `!Configuration.containsKey(PropertyKey.SECURITY_LOGIN_USERNAME)` at the end of the test